### PR TITLE
fix: corrected install instructions and added getRawValue()

### LINKED ISSUE #52: Incorrect README install instructions
In your README "Getting Started" section it says to use the command `npm install --save-dev ngx-forms-typed`, but the npm page sidebar says `npm i ngx-forms-typed`. I assume it's the latter, but you might want to get those in sync.

### PR DESCRIPTION
**Pull request by Josh015**
- add the support for `getRawValue()` on `TypedFormGroup`
- updated README with correct install instructions

Closes #52 #55

### CODE REVIEW

File: projects/forms/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 -   missing support for `ValidatorFn[]` and `AsyncValidatorFn[]` for `typedFormControl`
+-   missing support for `getRawValue()` on `TypedFormGroup`
```
Comment (gparlakov) [inline projects/forms/CHANGELOG.md:13]: Version 1.1.2 has already been released. I guess the [unreleased] above it is a bit misleading?

The following is what denotes the next version 1.1.3 provided there are only `chore:...` and `fix:...` commit messages:

```md
## [Unreleased]

## [1.1.3] - 2021-07-09

### Fixed

-   missing support for `getRawValue()` on `TypedFormGroup`
-   incorrect install instructions in README "Getting Started" section
```

**Review by gparlakov — COMMENTED** [review]
Thanks for persisting, thanks!

The commit messages are great.  

This is not a blocker for this particular PR, just FYI: the chore could be denoted with `chore:` or `build:`  `chore(README): corrected installation instructions`. Any one of these would be a signal for `semantic-release` that a new release is not needed.

**Comment by Josh015** [comment]
Okay, I've made the changes. Is there anything else I'm missing or are we good?

**Comment by gparlakov** [comment]
@Josh015 thanks!

**Comment by Josh015** [comment]
Hopefully once this and the license fix are deployed I can convince my team to let me integrate this package into our product.

**Comment by gparlakov** [comment]
The license is updated (though it took much more effort than expected) 
@Josh015 version 1.1.6 https://www.npmjs.com/package/ngx-forms-typed is licensed correctly. Good luck

**Pull request merged**